### PR TITLE
setup webpack with different environments

### PR DESCRIPTION
**Pull request by Arzamas-ka**
#### Description
Create package.json file and install React, Redux, React-Redux, React-Router, Jest. Install and configure Webpack & Babel to get build artifact by running npm command.  

Set DEV and PROD build configuration. Use env variables, dev server, optimizations for PROD build. Set up testing. You should have test command in your package.json file, which will run your future tests. Don’t use any React boilerplate (like create-react-app) for this task. 
 

#### Evaluation criteria
2 - Installed React, Redux, React-Redux, React-Router, Jest; 
3 - Configured Webpack; 
4 - Configured Babel.  Configured tests script; 
5 - Have DEV and PROD build commands (use env variables). 

#### jest
image

#### build:dev
image

#### build:prod
image

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import '../styles/index.css';
+
+const App = () => {
+
+  // comment for remove-comments-loader
```
[the remark this answers is not in the document]
Comment (Arzamas-ka) [reply src/components/App.js]: done

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
```
[the remark this answers is not in the document]
Comment (Arzamas-ka) [reply src/components/App.js]: done

**Comment by Arzamas-ka** [comment]
In addition, defined a config for favicon

**Review by osonovaakylai — APPROVED** [review]

**Comment by osonovaakylai** [comment]
According to task 2 packages like Redux, React-Redux, React-Router should be installed, please include them into dependencies

**Comment by Arzamas-ka** [comment]
> According to task 2 packages like Redux, React-Redux, React-Router should be installed, please include them into dependencies

the packages have been added

**Review by osonovaakylai — APPROVED** [review]

**Pull request merged**